includes/Classifai/Providers/Azure/ComputerVision.php

```diff
@@ -334,6 +331,39 @@ public function attachment_pdf_data_meta_box( $post ) {
 		<?php
 	}
 
+	/**
+	 * Callback to get the status of the PDF read with AJAX support.
+	 *
+	 * @param  int $attachment_id The attachment ID.
+	 * @return array Read and running status.
+	 */
+	public static function get_read_status( $attachment_id = null ) {
```

Comment:
@TylerB24890 I suggest creating another function to handle ajax requests instead of handling two types of requests.

Reply:
Sounds good. All set!